docs/en/latest/plugins/http-dubbo.md

```diff
@@ -0,0 +1,127 @@
+# limitations under the License.
+#
+-->
+
+## Description
+
+The `http-dubbo` plugin can encode HTTP requests into Dubbo protocol and forward them to upstream services (Note: in
+Dubbo 2.x, the serialization type of the upstream service must be fastjson).
+
+## Attributes
+
+| Name                     | Type    | Required | Default | Valid values | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+|--------------------------|---------|----------|---------|--------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| service_name             | string  | True     |         |              | Dubbo service name                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| service_version          | string  | False    | 0.0.0   |              | Dubbo service version, default 0.0.0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
```suggestion
| service_version          | string  | False    | 0.0.0   |              | Dubbo service version                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

this is sufficient since the default value is listed in the default column